src/ray/gcs/gcs_server/gcs_placement_group_scheduler.h

```diff
@@ -590,6 +592,10 @@ class GcsPlacementGroupScheduler : public GcsPlacementGroupSchedulerInterface {
 
   /// The nodes which are releasing unused bundles.
   absl::flat_hash_set<NodeID> nodes_of_releasing_unused_bundles_;
+
+  /// The syncer of resource. This is used to report placement group updates.
```

Comment:
I think not removing it is also fine? Since updating this using the existing path will make update slower.

I think one of potential way is to allow both ways (resource report update new resources & also pg actively updates it). But we can discuss it when we delete the logic

Reply:
I feel this is just too complicated. In the end, raylet will report this (convert it back) but it's taken care of by the syncer module.
I feel update/delete resources can just be a normal resource update.  With that, I feel things probably are cleaner. How do you think?